libs/blocks/media/media.css

```diff
@@ -265,6 +265,14 @@ div[class*="-up"] .media .foreground > .media-row {
   .media.large > .foreground .media-row .image {
     width: 50%;
   }
+
+  .media.medium-compact > .foreground .media-row .image {
+    width: 41.667%;
```

Comment:
Oddly specific number?

Reply:
Yeah it's (500/1200) 500px width of image / 1200px width of container . We had a whole discussion about whether they want it to be proportional or fixed, etc. and they want it to be proportional so that's what it shakes out to be.